### ISSUE TITLE
sv_msgsend.lua:32 in upvalue onsuccess Line 31 -> 36

### ISSUE THREAD

**Issue by Xer0n** [troit5ky/gmod_discord_relay#9]
This crashes the server, it's happened twice when someone joins with parenthesis in their name. 

Lua Stack:
[C][+0] [C] in string_find
[L][+1] addons/gmod_discord_relay/lua/relay/sv_msgsend.lua:32 in upvalue onsuccess Line 31 -> 36
[L][+2] lua/includes/modules/http.lua:29 in anonymous function Line 25 -> 31

Log:
[05:31:09] Client "Napalm (OOF)" connected.

--CRASH--

[05:32:18] New map: gm_genesis
[05:32:18] Client "C H 4 I2 Z" connected.
[05:32:23] Client "Xer0n_" connected.
[05:33:11] Client "Xer0n_" spawned in server <STEAM_0:1:36549678> (took 48 seconds).
[05:35:45] Client "C H 4 I2 Z" spawned in server <STEAM_0:0:17681186> (took 207 seconds).
[05:37:03] Dropped "C H 4 I2 Z" from server<STEAM_0:0:17681186>
[05:38:00] Client "Napalm (OOF)" connected.

--CRASH--

[05:39:09] New map: gm_genesis
[05:39:15] Server is shutting down/changing levels.

**Comment by Xer0n** [comment]
I just disabled the avatar lookup since we don't use those, doesn't crash when special characters join now.

**Comment by troit5ky** [comment]
Hello, could you please send me a link to them "Napalm (OOF)"?

**Comment by Xer0n** [comment]
I don't think that will help, I changed my name to Xer0n_ (Test) and it crashed the server.  I commented out everything that relied on the avatar lookup and it's working again, something about special characters in their name.

https://steamcommunity.com/id/Napalm_bomb
